discovery/spec.md

```diff
@@ -225,6 +225,7 @@ The following pseudo JSON shows the defintion of an Endpoint:
         "strict": true|false ?
     }, ?
 
+    "format": "STRING",
```

Comment:
Since Endpoints are also Groups, we needed to add "format" since it's a field in Groups